src/app/lib/utilities/srcSet/index.js

```diff
@@ -2,14 +2,36 @@ import buildIChefUrl from '#lib/utilities/ichefURL';
 
 const DEFAULT_RESOLUTIONS = [240, 320, 480, 624, 800];
 
+export const getMimeType = srcset => {
+  if (!srcset) return null;
+
+  const [firstSrcset] = srcset?.split(',');
```

Comment:
I believe the `?.` is in the wrong place here?  `srcset` is always defined (because of line 6), but `split` might not be (eg, if `srcset` is defined but not a string)

```suggestion
  const [firstSrcset] = srcset.split.?(',');
```

Although updating line 6 to ensure it is a string is probably cleaner